pkg/runtime/runtime.go

```diff
@@ -208,11 +205,34 @@ func newDaprRuntime(ctx context.Context,
 		Namespace:             namespace,
 	})
 
+	schedulerClients := clients.New(clients.Options{
+		Addresses: runtimeConfig.schedulerAddress,
+		Security:  sec,
+		Healthz:   runtimeConfig.healthz,
+	})
+
+	actors := actors.New(actors.Options{
```

Comment:
Should `New` return an err or should we check for the actors `disabled` error after creating the `actors` variable, but before using it below? This way we can err out earlier rather than later on? At this point the caller (dapr) isn't aware of the error because its stored in the `disabled` field. Its only an issue if `len(placementAddresses) == 0`.